data/json/mutations/mutations.json

```
@@ -2534,6 +2534,17 @@
     "threshreq": [ "THRESH_CEPHALOPOD" ],
     "category": [ "CEPHALOPOD" ]
   },
+  {
+    "type": "mutation",
+    "id": "INK_GLANDS",
+    "name": "Ink glands",
+    "points": 1,
+    "visibility": 1,
+    "ugliness": 1,
+    "description": "You noticed a several ink glands on your torso. These will spray defensive ink and could blind an attacker in an emergency. The torso must not be covered.",
```

```suggestion
    "description": "Several ink glands have grown onto your torso. They can be used to spray defensive ink and blind an attacker in an emergency, as long as the torso isn't covered.",
```
makes it clear that it has to be manually activated

---

Thank you for the edit the original description was pretty bad 👍 I actually wished for someone to fix it when I wrote it 😄